addons/website/models/res_config_settings.py

```diff
@@ -169,6 +170,15 @@ def action_open_robots(self):
             'target': 'new',
         }
 
+    def action_ping_sitemap(self):
+        if not self.website_id._get_http_domain():
+            raise Warning(_("You don't have defined your domain"))
```

Comment:
I don't think that's correct english, maybe `There is no domain defined on your website`?